static/app/components/events/featureFlags/eventFeatureFlagList.tsx

```diff
@@ -0,0 +1,199 @@
+import {useCallback, useRef, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {CompactSelect} from 'sentry/components/compactSelect';
+import DropdownButton from 'sentry/components/dropdownButton';
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import {EventDataSection} from 'sentry/components/events/eventDataSection';
+import {FeatureFlagDrawer} from 'sentry/components/events/featureFlags/featureFlagDrawer';
```

Comment:
I think there are circular imports in here. It's not a build problem but something that maybe means the organization of stuff could be looked at.

Reply:
fixed